src/js/models/layer.js

```diff
@@ -134,12 +134,12 @@ define(function (require, exports, module) {
         /**
          * @type {Immutable.List.<Shadow>}
          */
```

Comment:
I think a default value should be the value that works so that it doesn't rely on the users (us) to set a valid default value. In this case, I found I have to search over the files to make sure I can safely assume the dropShadows/innerShadows attr is always a list. Any thoughts?